docs/diagrams/DeleteTeamSequence.puml

```diff
@@ -1,7 +1,7 @@
 @startuml
 !include style.puml
 skinparam ArrowFontStyle plain
-!define input "/delete_team module/CS2103T tutorial/T09 name/Team 1"
+!define input "/delete_team module/CS2103T tutorial/T09 tutorial/Team 1"
```

Comment:
Hi @Jajared, is there a typo here? it should be team instead?

Reply:
Yep should team my bad